packages/core/src/endowmentsToolkit.js

```diff
@@ -379,7 +484,7 @@ function endowmentsToolkit({
     endowmentSources.forEach((source) => {
       const descriptors = Object.getOwnPropertyDescriptors(source)
       Object.values(descriptors).forEach((desc) => {
-        if ('get' in desc) {
+        if ('get' in desc && desc.get) {
```

Comment:
I usually do `desc.get !== undefined`

Reply:
i want it to be truthy

Reply:
fwiw `'foo' in bar` is the TypeScript-Approved:tr: way to check if a prop "has" something; skipping right to `foo.bar` will typically be an error if it's unclear from type inference

Reply:
in the case of a descriptor there's no difference between it being truthy or not-undefined because it can only be a function or missing. 
Is this for typescript only?

Reply:
According to TS, `get` can be `null`.  And `get: null` would be true for `'get' in desc`; hence the truthiness check.  TS' ECMAScript libs aren't the spec, but I trust them.

Reply:
If we want to support the folks using typescript directly on platforms we don't yet foresee (poor souls) this seems like the right thing to do.